src/controllers/keystore/keystore.ts

```diff
@@ -605,15 +617,12 @@ export class KeystoreController extends EventEmitter {
     return JSON.stringify(keyBackup)
   }
 
-  /*
-    DOCS
-    keyAddress: string - the address of the key you want to export
-    publicKey: string - the public key, with which to asymmetrically encypt it (used for key sync with other device's keystoreId)
-  */
-  async exportKeyWithPublicKeyEncryption(
-    keyAddress: string,
-    publicKey: string
-  ): Promise<Encrypted> {
+  async sendPrivateKeyToUi(keyAddress: string) {
+    const decryptedPrivateKey = await this.#getPrivateKey(keyAddress)
+    this.#windowManager.sendWindowUiMessage({ privateKey: `0x${decryptedPrivateKey}` })
```

Comment:
@sonytooo please take a look at this mechanism

Reply:
yes, we synced with Bobi and this is exactly how we want it to be. I can't think of a more direct way of sending something from the controllers to the UI